.github/workflows/sync-to-aur.yml

```diff
@@ -0,0 +1,45 @@
+name: Sync package to AUR
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Save ssh key
+        run: |
+          mkdir -p ~/.ssh
+          echo ${{ secrets.AUR_SSH_KEY }} > ~/.ssh/id_rsa
+          chmod 600 ~/.ssh/id_rsa
```

Comment:
Let's use an `Ed25519` key, it's faster and smaller than RSA.

And you can do all this in a single step:
```
echo ${{ secrets.AUR_SSH_KEY }} | install -Dm400 /dev/stdin ~/.ssh/id_rsa
```